tests/test_check_new_assets.py

```diff
@@ -11,21 +11,31 @@
 
 @pytest.mark.parametrize("chain", get_substrate_chains(), ids=task_ids)
 class TestAssets:
-    # workaround to differences between asset names at runtime and in our configuration
-    asset_mapping = {
-        'AUSD': 'KUSD',
-        'XX': 'xx'
-    }
-    
+    # assets that have no working cases or having differences between
+    # asset names at runtime and in our configuration
+    exclusions = {
+        'Karura': {'AUSD': 'KUSD'},
+        'Acala': {'AUSD': 'KUSD'},
+        'Shiden': {'AUSD': 'KUSD'},
+        'Bifrost Kusama': {'AUSD': 'KUSD', 'DOT': ''},
+        'Kintsugi': {'AUSD': 'KUSD', 'INTR': '', 'iBTC': 'IBTC', 'DOT': ''},
+        'Basilisk': {'AUSD': 'KUSD'},
+        'Turing': {'AUSD': 'KUSD'},
+        'XX network': {'XX': 'xx'}}
```

Comment:
I would suggest not to mix exclusions and asset mapping in terms of keeping it separate as the reasons of doing that is separate too:

```suggestion
     # workaround to differences between asset names at runtime and in our configuration
    asset_mapping = {
        'AUSD': 'KUSD'
    }
    # assets that has no working cases on network
    exclusions = {
        'Bifrost Kusama': {'DOT': ''},
        'Kintsugi': {'INTR': '', 'IBTC': '', 'DOT': ''}}
```
Also we can do it smaller by remove XX token, as the difference only in register